spec/views/admin/users/index_html_slim_spec.rb

```diff
@@ -5,6 +5,7 @@
     responder = find_or_create :foi_responder
     approver = create :approver
     assign(:users, User.where(id: [responder.id, approver.id]).page(1).decorate)
+    assign(:active_users, User.where(id: [responder.id, approver.id]).page(1).decorate)
 
```

Comment:
was hoping for an assert that the user count is displayed correctly? Not sure about line 8 - in the controller, the user list isn't decorated